### PR TITLE
Add recipe for bitbag/mollie-plugin

### PR DESCRIPTION
**Pull request by kptLucek**
| Q             | A
| ------------- | ---
| License       | MIT
| Packagist     | [Link](https://packagist.org/packages/bitbag/mollie-plugin)

### CODE REVIEW

File: bitbag/mollie-plugin/3.5/config/packages/bitbag_mollie_plugin.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+imports:
+    - { resource: "@BitBagSyliusMolliePlugin/Resources/config/config.yaml" }
+
+winzou_state_machine:
+    sylius_order_checkout:
+        transitions:
+            complete:
+                from: [cart, addressed, shipping_selected, shipping_skipped, payment_selected, payment_skipped]
+                to: completed
```
Comment (Nyholm) [inline bitbag/mollie-plugin/3.5/config/packages/bitbag_mollie_plugin.yaml:9]: Why is this needed?

File: bitbag/mollie-plugin/3.5/manifest.json [hunk 2]
```diff
@@ -0,0 +1,35 @@
+{
+    "bundles": {
+        "BitBag\\SyliusMolliePlugin\\BitBagSyliusMolliePlugin": ["all"]
+    },
+    "copy-from-recipe": {
+        "config/": "%CONFIG_DIR%/"
+    },
+    "copy-from-package": {
+        "tests/Application/templates/bundles/SyliusAdminBundle/Order/Show/Summary/_item.html.twig": "templates/bundles/SyliusAdminBundle/Order/Show/Summary/_item.html.twig",
```
Comment (Nyholm) [inline bitbag/mollie-plugin/3.5/manifest.json:9]: Why do you copy things from other bundles?

**Comment by fabpot** [comment]
Closing as there is no feedback.
